src/Microsoft.ML.LightGBM/LightGbmMulticlassTrainer.cs

```diff
@@ -51,20 +51,15 @@ internal LightGbmMulticlassTrainer(IHostEnvironment env, LightGbmArguments args)
         /// <param name="numBoostRound">Number of iterations.</param>
         /// <param name="minDataPerLeaf">The minimal number of documents allowed in a leaf of the tree, out of the subsampled data.</param>
         /// <param name="learningRate">The learning rate.</param>
-        /// <param name="advancedSettings">A delegate to set more settings.
-        /// The settings here will override the ones provided in the direct signature,
-        /// if both are present and have different values.
-        /// The columns names, however need to be provided directly, not through the <paramref name="advancedSettings"/>.</param>
-        public LightGbmMulticlassTrainer(IHostEnvironment env,
+        internal LightGbmMulticlassTrainer(IHostEnvironment env,
             string labelColumn = DefaultColumnNames.Label,
```

Comment:
Same question here, regarding unifying the two constructors?

Reply:
#2100

---
In reply to: [251542646](https://github.com/dotnet/machinelearning/pull/2261#discussion_r251542646) [](ancestors = 251542646)